src/pages/en/guides/styling.md

```diff
@@ -140,9 +216,17 @@ h-tick {
 <h-tick>✓</h-tick>
 ```
 
-## Autoprefixer
+## PostCSS
+
+You can use Tailwind or any [PostCSS plugin](https://www.postcss.parts/) by adding a `postcss.config.cjs` file to the root of your project. Follow the documentation for the plugin you’re trying to install for configuration and setup.
+
+Be aware that this plugin will run on all CSS in your project, including any files that compiled to CSS (like `.scss` Sass files, for example).
+
+>💡 _CSS in `public/` **will not be transformed!** Instead, place it within `src/` if you’d like PostCSS to run over your styles._
 
-[Autoprefixer][autoprefixer] takes care of cross-browser CSS compatibility for you. Use it in astro by installing it (`npm install --save-dev autoprefixer`) and adding a `postcss.config.cjs` file to the root of your project:
+### Autoprefixer
```

Comment:
similar to Tailwind, I think this section can be removed as "too opinionated". If someone wants to add autoprefixer, then the docs above is enough to follow to add it.

If you want to keep a reference, then the PostCSS section could be rewritten to:

```
Astro supports [PostCSS](https://www.postcss.parts/) out of the box. To enable, add a `postcss.config.cjs` file to the root of your project and add your (like autoprefixer)  to the file. Astro will automatically run your styles through PostCSS.